core/scss/components/site-search/_site-search.scss

```diff
@@ -44,12 +44,6 @@
   right: 12px;
   width: auto;
 
-  @media #{$breakpoint-lg} {
```

Comment:
This was causing the item to be placed funny for me. I don't think it is needed any longer.